tutorials/polkadot-sdk/build-a-blockchain/build-a-local-blockchain.md

```diff
@@ -0,0 +1,174 @@
+# Build a Local Blockchain
+
+## Introduction
+
+[Polkadot-SDK](https://github.com/paritytech/polkadot-sdk){target=\_blank} offers a versatile and extensible blockchain development framework, enabling you to create custom blockchain nodes tailored to your specific application or business requirements. This tutorial guides you through the process of compiling and launching a single local blockchain node using the [Polkadot-SDK Solochain Template](https://github.com/paritytech/polkadot-sdk-solochain-template){target=\_blank}.
+
+The node template provides a pre-configured, functional single-node blockchain that you can run in your local development environment. It includes several key components, such as user accounts and account balances.
+
+These pre-defined elements allow you to experiment with common blockchain operations without requiring any initial modifications to the template.
+In this tutorial, you will:
+
+- Build and start a local blockchain node using the node template
+- Explore how to use a front-end interface to:
+    - View information about blockchain activity.
+    - Submit a transaction
+
+By the end of this tutorial, you'll have a working local blockchain and understand how to interact with it, setting the foundation for further customization and development.
+
+## Prerequisites
+
+To get started with the node template, you'll need to have a few things set up on your development machine first:
+
+- Rust Installation - the node template is written in [Rust](https://www.rust-lang.org/){target=\_blank}, so you'll need to have it installed and configured on your system. Refer to the [Installation]() section for step-by-step instructions on setting up your development environment
+
+## Compile a Node 
+
+The [Polkadot-SDK Solochain Template](https://github.com/paritytech/polkadot-sdk-solochain-template){target=\_blank} provides a ready-to-use development environment for building using the [Polkadot-SDK](https://github.com/paritytech/polkadot-sdk){target=\_blank}. Follow these steps to compile the node:
+
+1. Open a terminal on your computer
+
+2. Clone the node template repository:
```

Comment:
I believe that these two can be merged into one single step, like:

"In your terminal, clone the node template repository"

(Or even more directly, "clone the node template repository" and that's it)